--- sphinx3-0.1/configure.in.orig	Mon Dec  2 18:24:11 2002
+++ sphinx3-0.1/configure.in	Fri Jan 31 23:47:09 2003
@@ -36,10 +36,10 @@
 audio_test_file=""
 
 AC_ARG_WITH(alsa, [use ALSA library for sound I/O],
-            AC_CHECK_HEADER(sys/asoundlib.h,
+            [AC_CHECK_HEADER(sys/asoundlib.h,
                 [ad_libs="-lasound"; ad_files="ad_alsa.lo";
                  ad_backend="AD_BACKEND_ALSA"],
-                AC_ERROR(ALSA header file <sys/asoundlib.h> not found!)))
+                AC_ERROR(ALSA header file <sys/asoundlib.h> not found!))])
 
 if test "x$ad_files" = "x"; then
         case $host in
@@ -107,7 +107,6 @@
 AC_SUBST(ad_libs)
 AC_SUBST(ad_backend)
 AC_SUBST(audio_test_file)
-AC_SUBST(LIBOBJS)
 AC_SUBST(PERL)
 AC_SUBST(CSH)
 
